src/pages/checkout/CheckoutForm.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { states } from "./states";
+
+function CheckoutForm({
+  onStateChanged,
+}: {
+  onStateChanged: (e: React.FormEvent<HTMLSelectElement>) => void;
+}) {
+  return (
+    <>
+      <div className="fw-semibold fs-5">Shipping Details</div>
+      <div className="d-flex my-3">
+        <input
+          type="text"
+          name="first_name"
+          className="form-control me-3"
+          placeholder="First Name"
+          required
+        />
```

Comment:
I think it'd be nice to have some labels on these for accessibility's sake.

Generally you don't want to rely on the placeholder for the label. It should be more of an example than a label.

So here, you'd have
```
<label htmlFor="first_name">First Name</label>
<input
  id="first_name"
  placeholder="Maria"
  ...
/>